tests/foreman/installer/test_installer.py

```diff
@@ -1468,7 +1468,7 @@ def test_foreman_rails_cache_store(sat_non_default_install):
 
     :steps:
         1. Install Satellite.
-        2. Verify that foreman-redis package is installed.
+        2. Verify that foreman-redis package is not installed.
```

Comment:
Could we change the comment below `Verify foreman-rails-cache-store option works` to `does not work`. Also can we change the docstring and test name to reflect this is testing non-default cache. like `test_foreman_non_default_rails_cache_store`

Reply:
Actually, the installer option (foreman-rails-cache-store) has to work. It should allow users to install with non default cache. The default cache is what has been changed from "file" to "redis". I slightly updated comments and docstring. I hope it's better now. Please check.